shared/message/localisation/ResetRobotHypotheses.proto

```diff
@@ -23,14 +23,30 @@ package message.localisation;
 import "Matrix.proto";
 import "Vector.proto";
 
+/**
+ * Contains the robot's position hypotheses about itself.
+ */
 message ResetRobotHypotheses {
+
     message Self {
-        vec2   position     = 1;
-        mat2   position_cov = 2;
-        double heading      = 3;
-        double heading_var  = 4;
-        bool   absoluteYaw  = 5;
+
+        /**
+         * Estimate of the robot's {x, y} position.
```

Comment:
This is probably in field space?

Reply:
judging by this passage: 
```cpp
            ResetRobotHypotheses::Self leftSide;
            // Start on goal line
            leftSide.position << -fieldDescription.dimensions.field_length * 0.5,
                fieldDescription.dimensions.field_width / 2;
```
I think you're right. I'll add that to the comment.

Reply:
This is one of the main reasons why we have been trying to change our variable names to the form rSFf